test/e2e/node/kubelet.go

```diff
@@ -423,8 +423,8 @@ var _ = SIGDescribe("kubelet", func() {
 			})
 
 			AfterEach(func() {
-				framework.ExpectNoError(framework.DeletePodWithWait(f, c, pod), "AfterEach: Failed to delete pod ", pod.Name)
-				framework.ExpectNoError(framework.DeletePodWithWait(f, c, nfsServerPod), "AfterEach: Failed to delete pod ", nfsServerPod.Name)
+				framework.ExpectNoError(framework.DeletePodWithWait(f, c, pod), "AfterEach: Failed to delete client pod ", pod.Name)
+				framework.ExpectNoError(framework.DeletePodWithWait(f, c, nfsServerPod), "AfterEach: Failed to delete server pod ", nfsServerPod.Name)
```

Comment:
Should use either 
```
439    err := framework.DeletePodWithWait(f, c, pod)
440    Expect(err).To(HaveOccurred()
```
or
```
426    framework.ExpectNoError(framework.DeletePodWithWait(f, c, nfsServerPod), "AfterEach: Failed to delete server pod ", nfsServerPod.Name)
```

But not both.

Reply:
_but not both_: are you referring to my use of `Expect(...)` below?